README.md

```diff
@@ -1,2 +1,179 @@
-# metric-webapp
-React capstone project
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="./banner.webp" alt="logo" />
+  <br/>
+
+  <h1><b>Metric-WebApp</b></h1>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 \[Metric-WEBAPP\] ](#-metric-webapp-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
```

Comment:
- [ ] Also I could be better to make sure your basic project structure is organized in a logical way.   Kindly make take a look to the structure below to make it even better. 

```bash
app-root-directory
  |- package.json
  |- README.md
  |- /public
    |- index.html
  |- /src
    |- app.css
    |- index.js
    |- /components
    |- /container
    |- /pages
    |- /redux
  |- /node_modules
```

Reply:
fixed the project structure